components/support/sync-telemetry/docs/metrics.md

```diff
@@ -8,10 +8,40 @@ This means you might have to go searching through the dependency tree to get a f
 
 # Pings
 
+- [addresses-sync](#addresses-sync)
 - [bookmarks-sync](#bookmarks-sync)
+- [creditcards-sync](#creditcards-sync)
 - [history-sync](#history-sync)
 - [logins-sync](#logins-sync)
 - [sync](#sync)
+- [tabs-sync](#tabs-sync)
+
+## addresses-sync
+
+A ping sent for every Addresses engine sync. It doesn't include the `client_id` because it reports a hashed version of the user's Firefox Account ID.
+
+
+**Data reviews for this ping:**
+
+- <TODO>
```

Comment:
Don't we need to file this in?

Reply:
I'll update w/ links to the data review above.